go/vt/vttablet/tabletserver/repltracker/repltracker.go

```diff
@@ -88,6 +90,9 @@ func (rt *ReplTracker) MakeMaster() {
 		rt.hr.Close()
 		rt.hw.Open()
 	}
+	if rt.forceHeartbeat {
+		rt.hw.Open()
+	}
```

Comment:
when throttler runs, we need heartbeat writer to run. We do not need the reader because throttler reads directly from `_vt.heartbeat` table.

Reply:
Just a note that we could be calling Open and Close twice, but that is fine since they are idempotent.

Reply:
:+1: that was my understanding as well.